### PR TITLE
Use fontTools.unicodedata script(), block() in unicodeTools

### PR DESCRIPTION
**Pull request by moyogo**

### CODE REVIEW

**Comment by moyogo** [comment]
hmm, this may not be such a good idea for scripts since the order changes. For example Bopomofo and Inherited are now before Greek, or Unkown (if it isn’t skipped and appended last) and Coptic are before Cyrillic.

**Comment by anthrotype** [comment]
@moyogo what shall we do with this PR?

File: Lib/defcon/tools/unicodeTools.py [hunk 1]
```diff
@@ -2,2954 +2,7 @@
 from __future__ import unicode_literals
 import unicodedata
```
Comment (anthrotype) [inline Lib/defcon/tools/unicodeTools.py]: since you do `from fontTools import unicodedata` in the lines below, this line should be removed.

**Comment by anthrotype** [comment]
> this may not be such a good idea for scripts since the order changes. For example Bopomofo and Inherited are now before Greek, or Unkown (if it isn’t skipped and appended last) and Coptic are before Cyrillic.

and how is that a problem? if they appear first in unicode 11, they just do.
What is that `orderedScripts` variable used for, by the way?

**Comment by anthrotype** [comment]
i think this is good to merge.

**Pull request merged**